src/Centreon/Application/Webservice/TopologyWebservice.php

```diff
@@ -75,6 +93,82 @@ public function getGetTopologyByPage(): array
         return $result;
     }
 
+    /**
+     * @OA\Get(
+     *   path="/internal.php?object=centreon_topology&action=menuList",
+     *   description="Get list of menu items by acl",
+     *   tags={"centreon_topology"},
+     *   @OA\Parameter(
+     *       in="query",
+     *       name="object",
+     *       @OA\Schema(
+     *          type="string",
+     *          enum={"centreon_topology"},
+     *          default="centreon_topology"
+     *       ),
+     *       description="the name of the API object class",
+     *       required=true
+     *   ),
+     *   @OA\Parameter(
+     *       in="query",
+     *       name="action",
+     *       @OA\Schema(
+     *          type="string",
+     *          enum={"menuList"},
+     *          default="menuList"
+     *       ),
+     *       description="the name of the action in the API class",
+     *       required=true
+     *   ),
+     *   @OA\Parameter(
+     *       in="query",
+     *       name="reactOnly",
+     *       @OA\Schema(
+     *          type="integer"
+     *       ),
+     *       description="fetch react only list(value 1) or full list",
+     *       required=false
+     *   ),
+     *   @OA\Parameter(
+     *       in="query",
+     *       name="forActive",
+     *       @OA\Schema(
+     *          type="integer"
+     *       ),
+     *       description="represent values for active check",
+     *       required=false
+     *   )
+     * )
+     * @throws \RestBadRequestException
+     */
+    public function getMenuList()
+    {
+        $user = $this->getDi()[ServiceProvider::CENTREON_USER];
+
+        if (empty($user)) {
+            throw new \RestBadRequestException('User not found in session. Please relog.');
```

Comment:
shouldn't we add a log in login.log using centreonLog object -> to help support in their tasks ?